torch/_inductor/ir.py

```diff
@@ -4299,9 +4299,14 @@ def unflatten_args(new_tensor_args, new_non_tensor_args):
         # propagated the graph with, because for some operators running without a
         # constant would trigger an error / DataDependentException
         for x in tensor_args:
-            if x.get_name() in V.graph.constants:
+            # if x is a view of a constant, we need to realize the view
+            # (we can't pass the constant into the kernel directly)
+            if not isinstance(x, BaseView) and x.get_name() in V.graph.constants:
```

Comment:
Might be better to reconstruct view from the constant, but dont know if it really matters.

That was added because in some cases FakeTensor will have constant which was needed to not-graph break. 

e.g.

`tensor([0:torch.tensor(8)])`

But yea, dont think it matters here, unlikely to have view of 1-dim torch.tensor which is needing for indexing

Reply:
oh interesting...

lmk if you want me to try to get that supported in this PR, otherwise I'll probably just leave as-is

Reply:
i think it's probably fine